glyphs2fontir/src/toir.rs

```diff
@@ -0,0 +1,181 @@
+use std::collections::HashMap;
+
+use fontir::{
+    coords::{CoordConverter, DesignCoord, DesignLocation, NormalizedLocation, UserCoord},
+    error::Error,
+    ir,
+};
+use glyphs_reader::{Font, FontMaster};
+use ordered_float::OrderedFloat;
+
+fn design_location(axes: &[ir::Axis], master: &FontMaster) -> DesignLocation {
+    axes.iter()
+        .zip(master.axes_values.as_ref().unwrap())
+        .map(|(axis, pos)| (axis.tag.clone(), DesignCoord::new(pos.into_inner() as f32)))
+        .collect()
+}
+
+fn to_normalized(axes: &[ir::Axis], design_location: &DesignLocation) -> NormalizedLocation {
+    let axes: HashMap<_, _> = axes.iter().map(|a| (&a.tag, a)).collect();
+    design_location
+        .iter()
+        .map(|(tag, dc)| {
+            (
+                tag.clone(),
+                dc.to_normalized(&axes.get(tag).unwrap().converter),
+            )
+        })
+        .collect()
+}
+
+fn find_by_user_coord(
```

Comment:
should this be named `find_by_design_coord`? The `value` which is the key in this lookup is a DesignCoord, so it seems to me you are finding (the index of) the given design coordinate in the mappings array.

Reply:
Err yes. I think I got halfway to find_user_coord then changed my mind.